charts/controller/README.md

```diff
@@ -30,6 +30,16 @@ The command deploys Rookout on the Kubernetes cluster in the default configurati
 
 > **Tip**: List all releases using `helm list`
 
+## Installation without helm
+If you're not using helm tiller on your kubernetes, you'll still be able to install the controller. Helm will be needed to be installed locally just to create the yaml file from the templates.
```

Comment:
Suggested wording for the first part of the sentence:
"If you're not using helm with your kubernetes cluster"